src/cli/peakdetector/peakdetectorcli.cpp

```diff
@@ -785,6 +785,11 @@ void PeakDetectorCLI::writeReport(string setName,
 
         // try uploading to Polly
         QMap<QString, QString> creds = _readCredentialsFromXml(pollyArgs);
+        if (creds.empty()) {
+            _log->error() << "Error in credentials file. Exiting.." << std::flush;
```

Comment:
"Error in credentials file" seems like a more general error. If the file is empty, lets inform the user of that situation itself.

Reply:
Specific errors in credentials file are handled in `_readCredentialsFromXml` and logged in the cli log file. This is a message to the user before CLI exits.